stages/14-director-gui-check/gui-dmaas-device-aws

```diff
@@ -93,5 +92,7 @@ cat ~/.dmaas/aws_cred ~/.dmaas/minio_cred >> cred.ini
 
 bash scripts/dop-gui-stages/run-gui-tests.sh ${GROUP} ${GUID} ${URL} ${THREADS} eu-north-1
 
-# cp  $path/.awscluster2/config ~/.kube/config
-# bash $path/utils/e2e-cr jobname:tcid-dir-dmaas-schd-local-dev-aws-bucket jobphase:Completed
\ No newline at end of file
+cp  $path/.awscluster2/config ~/.kube/config
+
+cd ../..
+bash $path/utils/e2e-cr jobname:tcid-dir-dmaas-schd-local-dev-aws-bucket jobphase:Completed
```

Comment:
This line of code is not understood

Reply:
we are applying cr to mark the current job status as completed, After this job is marked as Complete, next job will fetch the status and if it's completed then the current job will start it's execution